docs/blog/2019-08-14-strivectin-case-study/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Migrating from Magento Enterprise to Gatsby"
+date: 2019-08-14
+author: Thomas Slade
+tags:
+  - ecommerce
+  - case-studies
+  - marketing
+  - headless-cms
+image: "./images/strivectin.jpg"
+showImageInArticle: true
+---
+
+[Strivectin](https://www.strivectin.com), founded in 2002, sells lab-tested, anti-aging skincare products. They offer direct-to-consumer products online and are stocked in thousands of retailers globally. Their first site went live on Magento Enterprise in 2011. During this time, the customer base has grown more than 10x. This growth introduced technical obstacles. Site outages became a constant problem during marketing events and press releases. The site performance was sluggish—even when utilizing a CDN, browser cache, and Magento’s page cache.
```

Comment:
```suggestion
[StriVectin](https://www.strivectin.com), founded in 2002, sells lab-tested, anti-aging skincare products. They offer direct-to-consumer products online and are stocked in thousands of retailers globally. Their first site went live on Magento Enterprise in 2011. During this time, the customer base has grown more than 10x. This growth introduced technical obstacles. Site outages became a constant problem during marketing events and press releases. The site performance was sluggish—even when utilizing a CDN, browser cache, and Magento’s page cache.
```

It looks like this is capitalized elsewhere so probably needs it here too?

Reply:
Thanks for pointing those out :clap:. Updated in 046c737